### ISSUE TITLE
old pictures exif metadata missing?

### ISSUE THREAD

**Issue by stanelie** [photoprism/photoprism#1816]
**What does not work as expected?**
On a fresh install of Photoprism, I uploaded a bunch of old pictures from my library. The exif information is not available in Photoprism, even if my old library shows it (Lychee)

**How can we reproduce it?**
Should I send you a picture to test with?

1. import picture into photoprism
2. view picture in photoprism
3. clic on it to get the detailed view
4. see that there are no details available

**What behavior do you expect?**

I'd like to see the EXIF info

**Can you provide us with example files for testing or screenshots?**

Example picture
![2_picasa-5668378310821258642_1024x768](https://user-images.githubusercontent.com/2935029/146074755-8ecf62db-6a82-4fe7-976a-5049cf9e089b.jpg)

**What version you are using?**

PhotoPrism® 211210-2cb90e7e-Linux-x86_64

**Any other helpful information?**

**Comment by graciousgrey** [comment]
What metadata is missing exactly?

I checked the exif data of your image using exiftool and it only shows:

```
{
  "SourceFile": "exifmissing.jpg",
  "ExifToolVersion": 12.16,
  "FileName": "exifmissing.jpg",
  "Directory": ".",
  "FileSize": "585 KiB",
  "FileModifyDate": "2021:12:14 21:56:20+01:00",
  "FileAccessDate": "2021:12:14 21:56:37+01:00",
  "FileInodeChangeDate": "2021:12:14 21:56:26+01:00",
  "FilePermissions": "rw-rw-r--",
  "FileType": "JPEG",
  "FileTypeExtension": "jpg",
  "MIMEType": "image/jpeg",
  "JFIFVersion": 1.01,
  "ResolutionUnit": "inches",
  "XResolution": 96,
  "YResolution": 96,
  "Comment": "CREATOR: gd-jpeg v1.0 (using IJG JPEG v62), quality = 100\n",
  "ImageWidth": 1024,
  "ImageHeight": 768,
  "EncodingProcess": "Baseline DCT, Huffman coding",
  "BitsPerSample": 8,
  "ColorComponents": 3,
  "YCbCrSubSampling": "YCbCr4:4:4 (1 1)",
  "ImageSize": "1024x768",
  "Megapixels": 0.786
```

**Comment by stanelie** [comment]
Here is what I see in Lychee :
![Screen Shot 2021-12-14 at 4 08 26 PM](https://user-images.githubusercontent.com/2935029/146079897-f198c7f8-dc32-4afd-8f67-1dbd8c4ee8cf.png)
Capture date, camera model, focal length...

Here is what I see in Photoprism :
![Screen Shot 2021-12-14 at 4 14 55 PM](https://user-images.githubusercontent.com/2935029/146080390-f0918087-3a6d-47a9-950a-14c0cc29b5dc.png)

**Comment by lastzero** [comment]
What do you see in Exiftool?

**Comment by stanelie** [comment]
From within the docker container, running exiftool on the example picture got the exact same info as @graciousgrey .

Strangely, I ran exiftool on the photo on the Lychee server itself, and I got the same missing info. So, something else is going on I think. Maybe Lychee stripped the exif info after writing it to its own database (and maybe transcoding the file)??

I will investigate further. Thanks for your time.
